tests/std/tests/P0896R4_views_take_while/test.cpp

```diff
@@ -258,6 +258,91 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
         }
     }
 
+#if _HAS_CXX23
+    using ranges::const_iterator_t, ranges::const_sentinel_t, ranges::cbegin, ranges::cend;
+
+    // Validate view_interface::cbegin
+    STATIC_ASSERT(CanMemberCBegin<R>);
+    STATIC_ASSERT(CanMemberCBegin<const R> == ranges::range<const R>
+                  && indirect_unary_predicate<const Pred, const_iterator_t<const V>>);

```

Comment:
In general, we should always `assert` and `static_assert` X and Y separately, instead of `X && Y`. In the event that the assertion fails, separate assertions make it clear which part failed. Occurs below on 303-304.